src/labels/CustomLabels.labels

```diff
@@ -7327,6 +7519,422 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of Track Donation checklist item in Get Started for End Users</shortDescription>
+        <value>Campaigns let you to track your constituent outreach in Salesforce. Use the Primary Campaign field on Opportunities to better evaluate your fundraising efforts.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of Track Donation checklist item in Get Started for End Users</shortDescription>
+        <value>Learn more about Campaigns in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of Track Donation checklist item in Get Started for End Users</shortDescription>
+        <value>Track Constituent Outreach</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of Track Donation checklist item in Get Started for End Users</shortDescription>
+        <value>Track Donation Sources with Campaigns</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of Track Donation checklist in Get Started for End Users</shortDescription>
+        <value>In NPSP, you track donations, grants, pledges, and other types of revenue using the Opportunity object. You can create donations from an individual or from an organization such as a foundation, business, or government entity.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistTrackDonationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of Track Donation checklist in Get Started for End Users</shortDescription>
+        <value>Track Donations and Other Income Types with NPSP</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemExtraTwoMinutesVideo</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Extra Info in the Item &apos;2 Minutes Video&apos;</shortDescription>
+        <value>2 Minutes Video</value>
```

Comment:
Please change this to "2 Minute Video".

Reply:
@diego-caballero Sarah's requested change above is still pending.

Reply:
Ignore this.